src/components/Label/Label.tsx

```diff
@@ -8,6 +8,9 @@ interface LabelProps {
   htmlFor: string;
   /** ID that other components can cross reference for accessibility purposes */
   id?: string;
+  /** Controls whether the label should be inline with the input it goes with.
+   * This prop should only be used internally. */
+  isInlined?: boolean;
```

Comment:
Since this is only for internal use, remove it from Storybook. In `Label.stories.tsx`, add `isInlined: { table: { disable: true } },` to the `argTypes`.